monitor_ingest/README.md

```diff
@@ -0,0 +1,152 @@
+    git clone --branch st2 https://github.com/maximo-developer/monitor_ingest.git
+    ```
+## Get credentials
+2. Set credentials to connect Watson Analytics Service:
+- On the user interface, go to the Services tab.
+- Select Watson IoT Platform Analytics and click View Details.
+- In the Environment Variables field, click Copy to Clipboard.
+
+## Configuration
+
+3. Copy the example configuration in [monitor_ingest.yaml.example](./monitor_ingest.yaml.example)
+to `/opt/stackstorm/configs/monitor_ingest.yaml` and edit as per credentials copied in Step 2.
+
+Config file must have action_type, data_file_path and credentials keys.
+To run a specific Actions use `action_type`, `data_file_path` and `credentials` keys.
+`action_type` : action type name
+`data_file_path` : source json input file for specified action type
+`credentials` : Analytics service credentials
+ 
+** Check [config.schema.yaml](./config.schema.yaml) schema before creating your config file.
+
+Example configuration:
+
+```yaml
+action_type : "SetupEntityAction | SetupAddConstants | SetupAddDimesions | SetupAddFunctions "
+entity_name : ""
+data_file_path : "/opt/stackstorm/packs/monitor_ingest/etc/sample_usage_data.json"
+credentials:
+    _id: "id*******"
+    tenantId: "tenant Id"
+    db2:
+      username: "username"
+      password: "pwd"
+      databaseName: BLUDB
+      port: 50001
+      httpsUrl: "https://*******.cloud.ibm.com"
+      host: "*******.cloud.ibm.com"
+      security: SSL
+    iotp:
+      url: "https://orgId**.internetofthings.ibmcloud.com/api/v0002"
+      orgId: "org Id"
+      host: "orgId**.messaging.internetofthings.ibmcloud.com"
+      port: 8883
+      asHost: "*********.internetofthings.ibmcloud.com"
+      apiKey: "api_key"
+      apiToken: "api_token"
+    messageHub:
+      brokers:
+      - broker-0-****************************.us-south.eventstreams.cloud.ibm.com:9093
+      - broker-1-****************************.us-south.eventstreams.cloud.ibm.com:9093
+      - broker-5-****************************.us-south.eventstreams.cloud.ibm.com:9093
+      - broker-4-****************************.us-south.eventstreams.cloud.ibm.com:9093
+      - broker-2-****************************.us-south.eventstreams.cloud.ibm.com:9093
+      - broker-3-****************************.us-south.eventstreams.cloud.ibm.com:9093
+      username: "user name"
+      password: "password"
+    objectStorage:
+      region: global
+      username: "user name"
+      password: "password"
+    config:
+      objectStorageEndpoint: https://undefined
+      bos_logs_bucket: "log bucket"
+      bos_runtime_bucket: "runtime bucket"
+      mh_topic_analytics_alerts: ''
+```
+
+**Note** : When modifying the configuration in `/opt/stackstorm/configs/` please
+           remember to tell StackStorm to load these new values by running
+           `sudo st2ctl reload --register-configs`
+
+## Creating Vitual environment in StackStorm (Required only if testing in St2 env)
```

Comment:
This should happen automatically with `st2 pack install`, no need for these instructions